pkg/controller/scheduler.go

```diff
@@ -248,6 +248,16 @@ func (c *Controller) advanceCanary(name string, namespace string) {
 		return
 	}
 
+	// scale down the replicas to 0 after the service is pointing to the primary target
```

Comment:
```suggestion
	// scale down the canary deployment to 0 replicas after the service is pointing to the primary target
```

Reply:
since the target can also be a `DaemonSet`, how do you feel if the comment was changed to `scale down the canary target to 0 replicas after the service is pointing to the primary target`?

Reply:
works